CHANGELOG.rst

```diff
@@ -1,5 +1,13 @@
 Changelog for Pulse3D
 =====================
+0.32.2 (2020-03-23)
+-------------------
+
+Fixed:
+^^^^^^
+- Fixed missing wells when reading metadata for optical files.
```

Comment:
This should say something more along the lines of not trying to grab metadata from wells that aren't present when loading from optical files